calour/ms1_experiment.py

```diff
@@ -68,9 +68,8 @@ class MS1Experiment(Experiment):
     --------
     Experiment
     '''
-    def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs)
-        self.heatmap_databases = ('gnps',)
```

Comment:
i think should keep this database by default for ms (unless you set it somewhere else?)